spec/models/incoming_mail_handler_spec.rb

```diff
@@ -1,9 +1,8 @@
 describe "Handles incoming email" do
```

Comment:
the changes in this file are not purely from the master merge. I had an error in `Dispatcher` after merge where we were looking at `approved?` and then I realized we were still calling the `actions_for_approver` mailer, which should have been replaced with `StepMailer.proposal_notification` so I ended up changing it all at https://github.com/18F/C2/commit/e7a1c5e1fa0077190e7f1410e72dd1c82004c055 -- if you think it better to do that separately I can PR that commit on its own. 

you are right to think this smells funny